src/app/Command.cpp

```diff
@@ -31,19 +31,20 @@
 namespace chip {
 namespace app {
 
-CHIP_ERROR Command::Init(Messaging::ExchangeManager * apExchangeMgr)
+CHIP_ERROR Command::Init(Messaging::ExchangeManager * apExchangeMgr, InteractionModelDelegate * apDelegate)
```

Comment:
yes, p is the pointer, I think it can quickly inform user/implementer that it is pointer or not inside function, particular it has multiple similar variables in large function.

Reply:
offline chat with boris, and will keep this style here